BUILDING.md

```diff
@@ -4,14 +4,7 @@
 
 ### Prerequisites
 
-To build **mlkem-native**, you need `make` and a C90 compiler. To use the test scripts, you need Python3 with
-dependencies as specified in [requirements.txt](requirements.txt). We recommend using a virtual environment, e.g.:
-
-```bash
-python3 -m venv venv
-./venv/bin/python3 -m pip install -r requirements.txt
-source venv/bin/activate
-```
+To build **mlkem-native**, you need `make` and a C90 compiler. To use the test scripts, you need Python3 (>= 3.7).
```

Comment:
Have you tested older versions? Which features of 3.7 are we relying on?

Reply:
I have tested with Python 3.7 and it works fine. 

Python 3.6 fails because we are using the `capture_output` arugment for the subprocess.